app/api/crud.py

```diff
@@ -82,3 +82,29 @@ async def get(
         cross_node_results += response.json()
 
     return cross_node_results
+
+async def get_terms_labels(data_element_URI: str, labels: bool):
+    """Makes GET requests to one or more Neurobagel node APIs using httpx where the parameter is a Neurobagel variable."""
+
+    cross_node_results = []
+
+    for node_url in util.parse_nodes_as_list(util.NEUROBAGEL_NODES):
+        if labels:
+            url = node_url + "attributes/"+data_element_URI+"/vocab"
```

Comment:
```suggestion
            url = node_url + "attributes/" + data_element_URI + "/vocab"
```
can you check that you have flake8 setup and running in your IDE?